chain/client/src/stateless_validation/chunk_validator/orphan_witness_handling.rs

```diff
@@ -110,8 +110,6 @@ impl Client {
     ) {
         if let Some(signer) = signer {
             self.process_ready_orphan_witnesses(new_block, signer);
-        } else {
-            tracing::error!(target: "client", new_block=?new_block.hash(), "Cannot process ready orphan witnesses - not a validator");
```

Comment:
Maybe a debug log would be OK? A log saying why a witness was discarded could be useful for debugging.